src/sign/decentralized_party/signature_threshold_decryption_round.rs

```diff
@@ -156,22 +156,33 @@ where
         Ok((self.nonce_x_coordinate, signature_s))
     }
 
-    /// The lightweight $$ O(1) $$ threshold decryption logic, which simply verifies the output of
-    /// the decryption sent by the designated decrypting party. Blames it in case of an invalid
-    /// signature, and accepts otherwise.
+    /// A wrapper function for [`Self::verify_decrypted_signature_wrapper`] that uses self's
+    /// attributes to verify the decrypted signature.
     pub fn verify_decrypted_signature(
```

Comment:
```suggestion
    pub fn verify_decrypted_signature_in_session(
```
Perhaps this is a suitable name